lang/main.json

```diff
@@ -344,10 +344,11 @@
         "cancelPassword": "Cancel password",
         "conferenceURL": "Link:",
         "country": "Country",
-        "dialANumber": "To join your meeting, dial one of these numbers and then enter this PIN: __conferenceID__#",
+        "dialANumber": "To join your meeting, dial one of these numbers and then enter the pin.",
         "dialInConferenceID": "PIN:",
         "dialInNotSupported": "Sorry, dialing in is currently not suppported.",
```

Comment:
Could you fix the typo in "suppported "while you're in here?

Reply:
Fixed.